python/triton/runtime/autotuner.py

```diff
@@ -88,10 +89,35 @@ def _post_hook(args, exception):
         self.base_fn = fn
         while not inspect.isfunction(self.base_fn):
             self.base_fn = self.base_fn.fn
-        self.num_warmups = warmup
-        self.num_reps = rep
-        import torch
-        self.use_cuda_graph = use_cuda_graph and torch.cuda.is_available()
+
+        # If we got explicitly called via the old interface, raise a warning
+        # and proceed with the old behavior.
+        if warmup is not None or rep is not None or use_cuda_graph:
+            import warnings
+            warnings.warn("warmup, rep, and use_cuda_graph parameters are deprecated. See _ for details.",
```

Comment:
So do you plan to completely remove these arguments later?

Reply:
yep!